lib/runners/trace_writer.rb

```diff
@@ -110,5 +110,10 @@ def each_slice(string, size:)
         yield slice, truncated
       end
     end
+
+    def calc_duration_in_ms(start, finish)
+      seconds = finish - start
+      (seconds * 1000).truncate
```

Comment:
How about using `to_r` or `to_f` to get a more accurate duration?

```suggestion
      seconds = finish.to_r - start.to_r
      (seconds * 1000).truncate
```

Reply:
Perhaps, I think `Time#-` considers such accuracy internally... 🤔 

https://ruby-doc.org/core-2.7.2/Time.html#2D-method

Reply:
OK